src/constants/constants.js

```diff
@@ -0,0 +1,3 @@
+export const FILTER_ALL = "all";
```

Comment:
It can be one const, e.g.
export const FILTERS = {
ALL: 'all',
COMPLETED: 'completed',
'INCOMPLETE': 'incomplete'
}

Reply:
fixed